runelite-client/src/main/java/net/runelite/client/plugins/reorderprayers/ReorderPrayersPlugin.java

```diff
@@ -231,6 +232,15 @@ protected void shutDown() throws Exception
 		reorderPrayers();
 	}
 
+	@Subscribe
+	public void onGameStateChanged(GameStateChanged event)
+	{
+		if (event.getGameState() == GameState.LOGGED_IN)
```

Comment:
This only happens when you have prayer window opened right? so maybe add check here for it

Reply:
already gets checked in the reorder method